app/src/main/res/values/colors.xml

```diff
@@ -2,7 +2,8 @@
 <resources>
     <color name="colorPrimary">#3F51B5</color>
     <color name="colorPrimaryDark">#303F9F</color>
-    <color name="colorAccent">#FF4081</color>
+    <color name="colorAccent">#E65100</color>
     <color name="blackText">#000000</color>
     <color name="whiteBackground">#FFFFFF</color>
+    <color name="BEBEBE">#FFFFFF</color>
```

Comment:
`@color/BEBEBE` don't mean anything for me

Reply:
Name doesn't mean anything and value is equal to `whiteBackground`.